src/Repository/JobsRepository.php

```diff
@@ -16,27 +16,132 @@
  */
 class JobsRepository extends ServiceEntityRepository
 {
+    /**
+     * @param ManagerRegistry $registry
+     */
     public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Jobs::class);
     }
 
-    public function save(Jobs $entity, bool $flush = false): void
+    /**
+     * @param Jobs $entity
+     * @return void
+     */
+    public function save(Jobs $entity): void
     {
         $this->getEntityManager()->persist($entity);
-
-        if ($flush) {
-            $this->getEntityManager()->flush();
-        }
+        $this->getEntityManager()->flush();
     }
 
-    public function remove(Jobs $entity, bool $flush = false): void
+    /**
+     * @param Jobs $entity
+     * @return void
+     */
+    public function remove(Jobs $entity): void
     {
         $this->getEntityManager()->remove($entity);
+        $this->getEntityManager()->flush();
+    }
 
-        if ($flush) {
-            $this->getEntityManager()->flush();
+    /**
+     * @param int $id
+     * @return string
+     */
+    public function removeById(int $id): string
+    {
+        $listOfJobs = $this->findAll();
```

Comment:
use find($id) instead of findAll(). We need only one record with id = $id and find does exactly that. Avoid selecting all records as it is the main source of timeout error. When not possible use pagination (limit). In our case we select by id which is the primary key and it is unique. We will have 1 or 0 results.